### PR TITLE
Add explanation for multiple namespace installation

### PR DESCRIPTION
**Pull request by mtyazici**

### CODE REVIEW

File: docs/modules/ROOT/pages/get-started.adoc [hunk 1]
```diff
@@ -44,6 +44,16 @@ Then, use Kustomize to apply `bundle.yaml` with the changed namespace.
 kubectl apply -k .
 ----
 
+If you choose two deploy multiple Hazelcast Operators in different namespaces, you will need to update the name of every operator. You can add `namePrefix` field in the `kustomization.yaml` to differentiate them.
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/get-started.adoc:47]:
```suggestion
If you choose to deploy multiple Hazelcast Operators in different namespaces, you will need to update the name of every operator. You can add `namePrefix` field in the `kustomization.yaml` to differentiate them.
```

File: docs/modules/ROOT/pages/get-started.adoc [hunk 2]
```diff
@@ -44,6 +44,16 @@ Then, use Kustomize to apply `bundle.yaml` with the changed namespace.
...
+
+[source,yaml]
+----
+namespace: <your-namespace>
+resources:
+- bundle.yaml
+namePrefix: <operator-prefix>
```
Comment (hasancelik) [inline docs/modules/ROOT/pages/get-started.adoc:54]: I think suffix is better for readability and follow-up.
https://github.com/kubernetes-sigs/kustomize/blob/master/examples/transformerconfigs/README.md#prefixsuffix-transformer

**Review by hasancelik — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/get-started.adoc (hunk 2 above)
Comment (mtyazici) [reply docs/modules/ROOT/pages/get-started.adoc:54]: Sure I will change it

**Review by hasancelik — APPROVED** [review]
LGTM 👍

**Pull request merged**